package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "smartling-api-sdk-nodejs",
-  "version": "2.17.0",
+  "version": "2.17.1",
```

Comment:
It's not bug fix, it's adding new feature with no breaking changes, so should be `2.18.0` I believe.

Reply:
changed